toml/src/main/kotlin/org/rust/toml/crates/local/CratesLocalIndexCachesInvalidator.kt

```diff
@@ -0,0 +1,15 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.toml.crates.local
+
+import com.intellij.ide.caches.CachesInvalidator
+import com.intellij.openapi.components.service
+
+class CratesLocalIndexCachesInvalidator : CachesInvalidator() {
+    override fun invalidateCaches() {
+        service<CratesLocalIndexService>().invalidateCaches()
```

Comment:
Looks like `serviceIfCreated` should be used instead

Reply:
Sorry, I didn't really understand why